persistence/schema/gov.go

```diff
@@ -3,440 +3,151 @@ package schema
+	//TODO (@deblasis): Fix this
+	return fmt.Sprintf(`SELECT value FROM %s WHERE name='%s' AND height<=%d and enabled=true order by height desc limit 1`, ParamsTableName, paramName, DefaultBigInt)
+}
 
-		params.MessageStakeAppFee,
-		params.MessageEditStakeAppFee,
-		params.MessageUnstakeAppFee,
-		params.MessagePauseAppFee,
-		params.MessageUnpauseAppFee,
-		params.MessageStakeValidatorFee,
-		params.MessageEditStakeValidatorFee,
-		params.MessageUnstakeValidatorFee,
-		params.MessagePauseValidatorFee,
-		params.MessageUnpauseValidatorFee,
-		params.MessageStakeServiceNodeFee,
-		params.MessageEditStakeServiceNodeFee,
-		params.MessageUnstakeServiceNodeFee,
-		params.MessagePauseServiceNodeFee,
-		params.MessageUnpauseServiceNodeFee,
-		params.MessageChangeParameterFee,
+func NullifyParamQuery(paramName string, height int64) string {
+	//TODO (@deblasis): Fix this
+	return fmt.Sprintf(`UPDATE %s SET height=%d WHERE name='%s' AND height=%d`, ParamsTableName, height, paramName, DefaultBigInt)
+}
 
-		hex.EncodeToString(params.AclOwner),
-		hex.EncodeToString(params.BlocksPerSessionOwner),
-		hex.EncodeToString(params.AppMinimumStakeOwner),
-		hex.EncodeToString(params.AppMaxChainsOwner),
-		hex.EncodeToString(params.AppBaselineStakeRateOwner),
-		hex.EncodeToString(params.AppStakingAdjustmentOwner),
-		hex.EncodeToString(params.AppUnstakingBlocksOwner),
-		hex.EncodeToString(params.AppMinimumPauseBlocksOwner),
-		hex.EncodeToString(params.AppMaxPausedBlocksOwner),
+type ParamTypes interface {
+	int | int32 | int64 | []byte | string
+}
 
-		hex.EncodeToString(params.ServiceNodeMinimumStakeOwner),
-		hex.EncodeToString(params.ServiceNodeMaxChainsOwner),
-		hex.EncodeToString(params.ServiceNodeUnstakingBlocksOwner),
-		hex.EncodeToString(params.ServiceNodeMinimumPauseBlocksOwner),
-		hex.EncodeToString(params.ServiceNodeMaxPausedBlocksOwner),
-		hex.EncodeToString(params.ServiceNodesPerSessionOwner),
-		hex.EncodeToString(params.FishermanMinimumStakeOwner),
-		hex.EncodeToString(params.FishermanMaxChainsOwner),
-		hex.EncodeToString(params.FishermanUnstakingBlocksOwner),
-		hex.EncodeToString(params.FishermanMinimumPauseBlocksOwner),
-		hex.EncodeToString(params.FishermanMaxPausedBlocksOwner),
-		hex.EncodeToString(params.ValidatorMinimumStakeOwner),
-		hex.EncodeToString(params.ValidatorUnstakingBlocksOwner),
-		hex.EncodeToString(params.ValidatorMinimumPauseBlocksOwner),
-		hex.EncodeToString(params.ValidatorMaxPausedBlocksOwner),
-		hex.EncodeToString(params.ValidatorMaximumMissedBlocksOwner),
-		hex.EncodeToString(params.ValidatorMaxEvidenceAgeInBlocksOwner),
-		hex.EncodeToString(params.ProposerPercentageOfFeesOwner),
-		hex.EncodeToString(params.MissedBlocksBurnPercentageOwner),
-		hex.EncodeToString(params.DoubleSignBurnPercentageOwner),
+func SetParamQuery[T ParamTypes](paramName string, paramValue T, height int64) string {
+	//TODO (@deblasis): Fix this
+	fields := "name,height,value,enabled,type"
```

Comment:
We'll nned to generalized this since `enabled` will exist for `flags` but not for `params`.